cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/repositories_auth.go

```diff
@@ -31,12 +31,16 @@ const (
 	SecretCaKey         = "ca.crt"
 	SecretAuthHeaderKey = "authorizationHeader"
 	DockerConfigJsonKey = ".dockerconfigjson"
+
```

Comment:
may I suggest  "app.kubernetes.io/managed-by" for the key (ref https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels).  Seemed fitting to me so I was planning to use that in flux plugin

Reply:
i do not remember what we discussion we previously had, but this is following what is in the carvel plugin.
if we decide to change, we would either have different implementations, or need what we have in carvel.